agreements/project-contribution/fiscal-sponsorship-agreement.md

```diff
@@ -0,0 +1,277 @@
+Such fees, interest, and charges are necessary to compensate the Sponsor for its services in administering the Project and thus become unrestricted rather than restricted assets when credited to the Sponsor’s general fund.
+
+**Fundraising Activities:**
+The Project may solicit gifts, contributions, and grants to the Sponsor identified as made in support of the Project Purposes.
+The Project will provide information and the text of the Project’s fundraising materials for the Sponsor’s review in advance of the use and distribution of these materials.
+The Signatories agree to make changes to fundraising materials as the Sponsor determines are necessary to ensure compliance with applicable laws.
+
+**Fundraising Independence:**
+Individuals and organizations working on the Project may independently raise funds and compensate contributors, provided such activities comply with applicable laws and do not conflict with this Agreement or the Sponsor’s policies.
+
+To avoid confusion, independent financial activities must be clearly distinguished from those managed by the Sponsor.
+Public communications about such activities must include disclaimers clarifying that these funds are not part of the fiscal sponsorship arrangement and are managed solely by the responsible individuals or organizations.
+
+The Sponsor bears no responsibility for the financial management or legal compliance of independently raised funds.
+Those managing such funds must ensure compliance with all applicable laws and regulations.
+
+**Transparency and Coordination:**
+While external fundraising and financial management are permitted, Signatories are encouraged to maintain transparency and coordinate with the Sponsor to ensure alignment with the Sponsor’s mission and the Project’s goals and purposes.
+This includes providing relevant updates on independent activities that may affect the Project’s reputation, deliverables, or relationship with the Sponsor.
+
+### Restricted Fund and Variance Power
+
+Beginning on the Effective Date, all monetary contributions to the Project will be placed in a restricted fund, used solely for the Project Purposes.
+
+**Expenditure Authority:**
+While the Sponsor will delegate certain expenditure authority to the Signatories as provided in this Agreement and the [Terms and Conditions][], the Sponsor retains the unilateral right to spend such funds to accomplish the Project Purposes as nearly as possible[^1] within the Sponsor’s sole judgment, subject to any more specific donor-imposed restrictions on the use of such assets.
+
+[^1]: This provision is meant to ensure that if it is no longer feasible for the funds to be spent for the Project Purposes (for example if a sponsored project ceases to operate), CF can use the funds for other projects without seeking approval of a court.
```

Comment:
CF? (CommonHaus Fondation I guess, but I wasn't defined anywhere)

Reply:
THAT WAS FAST. holy cow. I need to send you a prize. Good catch, will fix.